community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/requirements.txt

```diff
@@ -7,7 +7,7 @@ google-cloud-bigquery==3.11.3
 google-cloud-core==2.3.3
 google-cloud-secret-manager~=2.22
 google-cloud-storage==2.10.0
-google-cloud-tpu==1.10.0
+google-cloud-tpu==1.21.0
```

Comment:
Sadly, that will not be enough, we need to change requirements.txt in slurm-gcp repo and roll an image first.

Reply:
Just a note, slurm-gcp repo uses lower versions for following packages:
```
requests==2.31.0
PyYAML==6.0.1
```